### DISCUSSION TITLE
Filter Option for Core Group Availability in Polls

### DISCUSSION THREAD

**Discussion by iamericfletcher** [lukevella/rallly#1244] [Ideas]
Hello everyone,

I'm posting to share a challenge I've faced while scheduling an Advisory Council meeting. After exploring 50 date and time options across three months, I've realized the need for a filter feature would be helpful.

Currently, I'm targeting the availability of a core group of 5-6 attendees, while also considering the schedules of the remaining 9 attendees. It would be helpful to have a filter feature that allows me to:

1. View only the options where the core group has responded with a "yes" of "if need be"
2. Easily toggle the filter on and off to return to the full list of options

In my opinion, this feature would enhance my ability to review scheduling polls and scenarios with management, especially when dealing with meetings with large groups, core attendees, and numerous dates and times under consideration. 

What do you think?

Sincerely,
Eric

**Comment by lukevella** [comment]
Hey @iamericfletcher, I really like this idea. I see this as part of a bigger peace of work which is to offer a dedicated UI for organizers to look at the data of their poll. Currently, organizers share the voting UI with elevated privileges like being able to edit any participant but this limits us in many ways. What I can say for sure is that this is a high priority feature that is on our radar.

**Reply by iamericfletcher** [reply]
So happy to see this! This will be so helpful if/when implemented :) Would make for a really terrific feature for me (and really all assistants) when the time comes to reviewing options with our bosses!
